org.overte.Overte.yml

```diff
@@ -0,0 +1,70 @@
+app-id: org.overte.Overte
+runtime: org.kde.Platform
+runtime-version: '5.15-23.08'
+sdk: org.kde.Sdk
+# QtWebEngine is not part of the KDE runtime,
+# so we use this package, which is also maintained by KDE developers.
+base: io.qt.qtwebengine.BaseApp
```

Comment:
You forgot to add the baseapp cleanup.

Reply:
https://github.com/flathub/io.qt.qtwebengine.BaseApp?tab=readme-ov-file#cleanup

Reply:
Thanks, I missed that.